packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -452,7 +451,7 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
             onBlur={handleBlur}
           />
           <span className="Combobox__arrow" />
-        </div>
+        </TextFieldWrapper>
```

Comment:
Nothing related to this particular PR, I am just thinking if we will refactor `<span className="Combobox__arrow" />` into `trailingChildren` of `TextFieldWrapper`? @scurker

Reply:
Yes, that should be how it works.